src/docs/framework-integration/vue.md

```diff
@@ -14,19 +14,43 @@ contributors:
 
 In order to use the custom element library within the Vue app, the application must be modified to define the custom elements and to inform the Vue compiler which elements to ignore during compilation. This can all be done within the `main.js` file. For example:
 
+### Add the component(s) to the app dependencies
+Typically done with `npm install --save test-components`
+
+
+### Import the components into the 'main.js' file
+by 
+
+- importing the node module
+- telling Vue to ignore the custom element tags
+- binding their code to the window object
+
+See `// HERE` comments:
+
 ```tsx
 import Vue from 'vue';
 import App from './App.vue';
-import { defineCustomElements } from 'test-components/dist/loader';
+import { defineCustomElements } from 'test-components/dist/loader'; // HERE
 
 Vue.config.productionTip = false;
-Vue.config.ignoredElements = [/test-\w*/];
+Vue.config.ignoredElements = [/test-\w*/]; // HERE
 
-defineCustomElements(window);
+defineCustomElements(window); // HERE
```

Comment:
Personally I find these comments more distracting than helpful. There are only a few lines here; not much to explain.

Reply:
Glad for comments from a neutral bystander-- what makes sense to me does not always make sense to everyone.

I like the "HERE" tags because it highlights what gets added to the Vue file. It is not otherwise obvious where the file is modified. 

But I could instead put it in the text?

Reply:
Ah I see. Maybe more explanatory comments would help, like this one from the React integration:
```js
// test-component is the name of our made up Web Component that we have
// published to npm:
import { defineCustomElements } from 'test-components/dist/loader';
```

Reply:
Another improvement over how I had put it. Thank you again.